FeedStoreChallenge/ManagedObjects/ManagedCache+CoreDataProperties.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ManagedCache+CoreDataProperties.swift
+//  FeedStoreChallenge
+//
+//  Created by Vita Solomina on 2021-05-01.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+extension ManagedCache {
+	@nonobjc public class func fetchRequest() -> NSFetchRequest<ManagedCache> {
```

Comment:
This method is a `CoreDataFeedStore` implementation detail so it should not be public

Please make it `internal`.
```suggestion
	@nonobjc class func fetchRequest() -> NSFetchRequest<ManagedCache> {
```